detection-rules/paypal_invoice_abuse.yml

```diff
@@ -12,36 +12,82 @@ source: |
   and length(attachments) == 0
   and sender.email.domain.root_domain == "paypal.com"
   and not any(headers.hops, .authentication_results.dmarc == "fail")
-  and strings.ilike(body.html.display_text, "*seller note*")
+  and (
+    strings.ilike(body.html.display_text, "*seller note*")
+    or strings.ilike(body.html.display_text, "*Note from *")
+  )
   and (
     (
-      // phone number but not 800 number
-      regex.contains(body.html.inner_text,
-                     '[\s:,-]\+?\d{1,2}[\s:,-]\(?([2-7][0-9]{2}|80[1-9]|8[1-9][0-9])\)?[\s:,-]\d{3}[\s:,-]\d{4}\b'
+      // icontains a phone number
+      (
+        regex.icontains(strings.replace_confusables(body.current_thread.text),
+                        'note from.{0,}\+?(\d{1}.)?\(?\d{3}?\)?.\d{3}.?\d{4}.{0,}Transaction ID'
+        )
+        or regex.icontains(strings.replace_confusables(body.current_thread.text),
+                           'note from.{0,}\+\d{1,3}[0-9]{10}.{0,}Transaction ID'
+        )
+        or // +12028001238
```

Comment:
I thought we fixed these comments?

Reply:
hmm I thought we did too.

Reply:
Just modified it again, was this where we landed?